### PR TITLE
added section on facets to tables article

### PR DESCRIPTION
**Pull request by ychae**

### CODE REVIEW

**Comment by ychae** [comment]
#196

File: articles/tables.md [hunk 1]
```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
 {% endtab %}
 {% endtabs %}
 
+## Table Facets
+The faceted navigation on `Tables` can be used to help simplify your search without having to use SQL-like queries. To use table facets, navigate to a `Table` or a `File View`. For this example, we'll be using the [Synapse Table Demo](https://www.synapse.org/#!Synapse:syn3079449/tables/){:target="_blank"} found in the [Wondrous Research Example](https://www.synapse.org/#!Synapse:syn1901847/wiki/56044){:target="_blank"}.
```
Comment (kdaily) [inline articles/tables.md]: used to simplify

Comment (kdaily) [inline articles/tables.md]: bit confusing to link to existing table that users can't see all schema buttons or modify it

File: articles/tables.md [hunk 2]
```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
...
+
+### Set facets
+In order to use simple search, you must first set columns to be facets in the schema editor. Select **Schema** in the upper right of your table and click on **Edit Schema**. In the resulting pop-up, select **Values** or **Range** from the dropdowns under the **Facet** option.
```
Comment (kdaily) [inline articles/tables.md]: simple vs advanced search modes

Comment (kdaily) [inline articles/tables.md]: define 'value' vs 'range' - which column types can be used for each - what to expect

File: articles/tables.md [hunk 3]
```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
...
+
+<img id="imageLg" src="/assets/images/set_facets.png">
+
+### See faceted (simple) search
+To see all the facets, click on **Show simple search** found above the SQL-query bar:
+
+<img id="imageLg" src="/assets/images/show_simple_search.png">
+
+#### Use simple search
+Select the features you're interested in to filter the table.
```
Comment (kdaily) [inline articles/tables.md]: no contractions

File: articles/tables.md [hunk 4]
```diff
@@ -681,6 +692,35 @@ Clicking on any file will download it.
...
+
+#### Toggling between simple and advanced search
+You can toggle from the simple search to the advanced search without losing the query results. For example, if in the simple search you've selected for treatmentArm `A`, age of `23:64`, and gender as `female`, the query will be preserved in the advanced search bar. 
+
+{% include note.html content="The slider for range in simple search is inclusive." %}
+
+<img id="imageSmall" src="/assets/images/simple_search_query.png">
+<img id="imageXL" src="/assets/images/query_statement_from_simple_search.png">
+
+However, when toggling back to simple search, the query will be reset.
```
Comment (kdaily) [inline articles/tables.md]: bold/emph on this!

**Pull request merged**